symplyphysics/laws/electricity/current_is_proportional_to_EMF.py

```diff
@@ -0,0 +1,33 @@
+
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+
+# Description
+## Ohm's law for the full section of the circuit for direct current is formulated: 
+## The current in the circuit is directly proportional to the sum of the EMF of the circuit and inversely
+## proportional to the sum of the resistances of the source and the circuit elements
+## law: I = EMF / (R + r)
+## Where I is the current flowing through circuit 
+## EMF - is electromotive force
+## R is impedance of this circuit
+## r is impedance of the power supply
+
+current = Symbol("current", units.current)
+EMF = Symbol("EMF", units.voltage)
```

Comment:
Don't shortcut. Use full descriptive names for symbols. Same for the law name - decipher EMF in law name.